.github/pull_request_template.md

```diff
@@ -0,0 +1,34 @@
+## PR Title :boom:
+
+Please title this PR with a summary of the change, preferably with the JIRA card number.
+
+Suggested format: __RHIROS-XXX | Title__
```

Comment:
```suggestion
Suggested format: __RHIROS-XXX | Title__
```

I prefer to use `-` instead of `|` and for all other repositories we are using `-` mark. can we support both here?

@upadhyeammit @saltgen  ^^ any thoughts?

Reply:
In my opinion, it would be good not to enforce a particular mark. I have seen many folks use `:`, `-` or `|` or white space in other RedHatInsights projects.

Reply:
By default the PR title is similar to commit message. 
I prefer to have consistent commit messages, so my choice will be keep any specific keyword instead of allowing multiple choices.

Reply:
I have posted a poll on this with the various styles folks prefer on the team channel

Reply:
I have updated the top two formats from the poll, for some amount of flexibility, thanks!

Reply:
Thank you @saltgen!